android/modules/ui/src/java/ti/modules/titanium/ui/NavigationWindowProxy.java

```diff
@@ -47,8 +50,8 @@ public void popToRootWindow(@Kroll.argument(optional = true) Object arg)
 	{
 		// Keep first "root" window
 		for (int i = windows.size() - 1; i > 0; i--) {
-			WindowProxy window = windows.get(i);
-			closeWindow(window, arg);
+			TiWindowProxy window = windows.get(i);
+			window.close(arg);
```

Comment:
There is a bug here.

You're not removing any of the objects from the `windows` member variables. Versus the old code would call `closeWindow()` which would call `windows.remove()`.

Reply:
My bad. I did not see that. Updated.